extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -89,16 +90,47 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
       }
       return choices;
     };
-    // If choices need to be shuffled, shuffle them, otherwise order the
+    // If choices need to be shuffled, shuffle them, if question is answered
+    // once before, get the previous order of choices, otherwise order the
     // choices based on their original index.
-    this.choices = (
-      showChoicesInShuffledOrder.value ? shuffleChoices(choicesWithIndex) :
-      choicesWithIndex.sort((c1, c2) => c1.originalIndex - c2.originalIndex));
+    this.questionIsOnceAnswered = this.isQuestionOnceAnswered();
```

Comment:
`questionIsOnceAnswered` -> `questionIsAnsweredOnce`
ditto here and elsewhere.

Reply:
Done!